boltdesignsystem/bolt
# Speed Up Monorepo Git Commit Hooks By ~90s

## Jira
N/A

## Summary
This PR switches the Bolt monorepo over to using [lint-staged](https://github.com/okonet/lint-staged) on any files being commited + disables the extra long running monorepo Jest test (which double-checks dependencies) until everything starts to run on Travis. 

Not only should this save us around 90 seconds of wait-time when trying to push but it should also automatically catch **_and fix_** any lint-related issues that can get automatically fixed when trying to commit!

## Details
I tested this out by updating a handful of files to confirm everything is working as expected... looks like there's a TON of Sass files we could automatically clean (but I'll save that for a separate PR).

## How to test
- Confirm Travis build working as expected
- Go ahead and try and try to commit something that Prettier / Eslint / Stylint would catch... 😉
